src/Resolvers/Microsoft.DotNet.SdkResolver/VSSettings.cs

```diff
@@ -34,24 +34,15 @@ private VSSettings()
                 return;
             }
 
-            string instanceId;
-            string installationVersion;
-            bool isPrerelease;
-
-            try
-            {
-                var configuration = new SetupConfiguration();
-                var instance = configuration.GetInstanceForCurrentProcess();
-
-                instanceId = instance.GetInstanceId();
-                installationVersion = instance.GetInstallationVersion();
-                isPrerelease = ((ISetupInstanceCatalog)instance).IsPrerelease();
-            }
-            catch (COMException)
```

Comment:
Hmm, there's another exception possible on pristine machine with No VS at all:

https://github.com/microsoft/vs-setup-samples/blob/a2d554616a9c82e77d5bb1b292417400732697e3/Setup.Configuration.CS/Program.cs#L47

Reply:
oh I remember that one too!

Reply:
I have addressed it, but I don't have a machine handy to test it right now.